src/redux/modules/fontFaces.js

```diff
@@ -9,13 +9,15 @@ export default function reducer(state = {}, action = {}) {
       const verses = action.result.entities.verses;
       const classNames = {};
 
-      Object.keys(verses).forEach((ayahId) => {
-        const verse = verses[ayahId];
+      if (verses !== undefined) {
```

Comment:
<img width="521" alt="screen shot 2017-03-26 at 7 25 09 pm" src="https://cloud.githubusercontent.com/assets/2888253/24331224/febbcfc6-1259-11e7-925f-6bf53057d001.png">

no, `if(!verses)` does not work. I think if we want to catch `null` and `undefined` value, we should use this one:

`if (verses !== undefined && verses !== null)`